### DISCUSSION TITLE
Link that takes you from table entry to another table with all associations

### DISCUSSION THREAD

**Discussion by odysseyskate** [Budibase/budibase#4898] [Help]
I have two data tables, 1 is a list of words, the 2nd is a list of words that are associated with table 1.

I have two views, one for each table. I click on "view" next to a table entry & I want it to take me to another table. This other table would show all entries on table 2 that are associated with the clicked item from table 1. Is this possible?

**Comment by mjashanks** [comment]
Hey @odysseyskate 

So, I think you are wanting to display a parent/child relationship, for example ...
- `One Project has many Tasks`
- On the `Project Detail` screen (`/projects/:id`), list the tasks assigned to that project.

So...

1. In the Project detail 
2. Click on the "Repeater" component in the left-hand panel
3. Add a new "Data Provider" component. This will place the data provider inside the repeater
4. Set the Data Provider's data source to `Relationship > Repeater.Projects.Tasks`
5. Add a table inside the new data provider

The table should now list all tasks belonging to that project.

Happy to explain how to link the screen up, if you need that as well.
